torch/_inductor/kernel/mm_common.py

```diff
@@ -336,6 +336,18 @@ def filtered_configs(
     {"config": (32, 256, 64, 6, 4), "cond": True},
 ]
 
+scaled_persistent_mm_kernel_configs = [
+    {"config": (128, 128, 64, 3, 8), "cond": True},
+    {"config": (128, 128, 128, 3, 8), "cond": True},
+    {"config": (128, 128, 128, 4, 8), "cond": True},
+    {"config": (128, 128, 128, 4, 4), "cond": True},
+    {"config": (128, 128, 128, 3, 4), "cond": True},
+    {"config": (128, 128, 128, 5, 4), "cond": True},
+    {"config": (128, 128, 128, 5, 8), "cond": True},
+    {"config": (128, 128, 128, 6, 8), "cond": True},
+    {"config": (128, 128, 64, 4, 8), "cond": True},
+]
```

Comment:
cc @drisspg: separated these configs used in `_scaled_mm` persistent+TMA template-based lowering, as ~half of them OOMs on SMEM for 2-byte dtypes. Kept the ones that don't in `persistent_mm_kernel_configs`.